src/database/queries.ts

```diff
@@ -100,3 +106,49 @@ export const getVouchersByVendorUuid = async (
     throw e;
   }
 };
+
+/**
+ * Create a new "vendor" document and at it to the vendors collection. 
+ */
+ export async function createVendor(email: string, name: string) {
+  try {
+    const docRef = await addDoc(vendorCollection, {
```

Comment:
After adding the document, make sure to also add the `uuid` field -- more details in this comment: https://github.com/calblueprint/vouchers4veggies/pull/10/files#r1001353362